apps/emqx/src/emqx_persistent_session_ds_router.erl

```diff
@@ -242,3 +260,30 @@ list_route_tab_topics() ->
 
 mria_route_tab_delete(Route) ->
     mria:dirty_delete_object(?PS_ROUTER_TAB, Route).
+
+stream(Tab = ?PS_ROUTER_TAB, MTopic) ->
```

Comment:
A comment describing the purpose and semantics of this function would be welcome.